dtmi/azureiot/paad/iotphone-1.json

```diff
@@ -0,0 +1,106 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:azureiot:PaaD:IoTPhone;1",
+  "@type": "Interface",
+  "displayName": "IoT Phone",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "AppTheme",
+      "writable": true,
+      "schema": {
+        "@type": "Enum",
+        "valueSchema": "string",
+        "enumValues": [
+          {
+            "name": "light",
+            "enumValue": "Light"
+          },
+          {
+            "enumValue": "Dark",
+            "name": "dark"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Command",
+      "name": "restartApp"
+    },
+    {
+      "@type": "Command",
+      "commandType": "synchronous",
+      "displayName": {
+        "en": "LightOn"
```

Comment:
should we remove the `en` form?

Reply:
> should we remove the `en` form?

nothing wrong with it

Reply:
My point is that the model should be consistent. Use this form everywhere, or not. Also, having also one locale is confusing. Should we assume `en` is the default? What's the behavior if is only set in some fields?

Reply:
agreed on consistency - just a thought: maybe we want to use this model to showcase how to localize a DTDL model correctly? Have strings for English, Spanish, German.

Reply:
I'd suggest to showcase DTDL localization capabilities with a different sample. We need to deep dive what central supports before adding strings that are not editable/visible.